pkg/minikube/node/cache.go

```diff
@@ -122,11 +122,19 @@ func beginDownloadKicBaseImage(g *errgroup.Group, cc *config.ClusterConfig, down
 	register.Reg.SetStep(register.PullingBaseImage)
 	out.Step(style.Pulling, "Pulling base image {{.kicVersion}} ...", out.V{"kicVersion": kic.Version})
 	g.Go(func() error {
+		images := make([]string, 0, 2+len(kic.FallbackImages))
 		baseImg := cc.KicBaseImage
+		baseFallbackImg := ""
 		if baseImg == kic.BaseImage && len(cc.KubernetesConfig.ImageRepository) != 0 {
-			baseImg = updateKicImageRepo(baseImg, cc.KubernetesConfig.ImageRepository)
+			baseImg, baseFallbackImg = updateKicImageRepo(baseImg, cc.KubernetesConfig.ImageRepository)
 			cc.KicBaseImage = baseImg
 		}
+		images = append(images, baseImg)
```

Comment:
if minikube does not start with `--image-registry` option, or specify a different kic image that without sha (with `--base-image` option), it will not add the image without sha into the fallback list